script.js

```diff
@@ -5,13 +5,42 @@ let cartElem = document.getElementById('cart');
 let menuElem = document.getElementById('header_menu');
 let bodyElem = document.getElementById('body');
 let cartListElem = document.getElementById('cart_list');
+let featuresElem = document.getElementById('button_submenu');
```

Comment:
Write the full word, no need to shorten it to `Elem`. It could otherwise potentially cause confusion if someone else had to work with your code.

Reply:
Fixed